azerothcore/azerothcore-wotlk
# feat(Core/Items): Delete old item id from recovery item

## Changes Proposed:

This PR proposes changes to:

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:

## Tests Performed:

This PR has been: I didn't test this code and i dont know if it works or not xD


## How to Test the Changes:



1. enable this option from world conf ItemDelete.Method = 1
2. sell your items to the vendor
3. and enable this option ItemDelete.KeepDays = 30, items should deleted after 30day from recovery_item

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
